packages/css/README.md

```diff
@@ -0,0 +1,53 @@
+# CSS definitions of the web platform
+
+This package contains CSS property definitions scraped from the latest versions of web platform specifications in [webref](https://github.com/w3c/webref), with fixes applied to ensure all CSS value definitions can be parsed and are internally consistent.
```

Comment:
That makes sense. There are tests that we could add over time to make sure that we have all the CSS properties and values that we need. Right now, there are values that are not used anywhere (e.g. because I believe that we do not yet extract @rules) and there are likely missing values (e.g. because these values are defined in prose).

Reply:
Indeed. I started writing some code for https://github.com/foolip/mdn-bcd-collector/issues/705 and it quickly became clear that what's extracted from CSS specs isn't consistent. There are duplicate definitions and missing definitions.